prisma/schema.prisma

```diff
@@ -30,7 +30,7 @@ model User {
   Signup   Signup[]
 
   createdAt DateTime  @default(now())
-  updatedAt DateTime? @updatedAt()
+  updatedAt DateTime?
```

Comment:
We need updatedAt() default value

Reply:
Keeping updatedAt as a default value will fill the updatedAt column during the creation of the user so it is omitted and updatedAt is setup to only fill during the patch/update operation.